multichain-testing/scripts/update-config.sh

```diff
@@ -16,17 +16,6 @@ sed -i -e 's/index_all_keys = false/index_all_keys = true/g' $CHAIN_DIR/config/c
 sed -i -e 's/seeds = ".*"/seeds = ""/g' $CHAIN_DIR/config/config.toml
 sed -i -e 's#cors_allowed_origins = \[\]#cors_allowed_origins = \["*"\]#g' $CHAIN_DIR/config/config.toml
 
-echo "Increase $(*_bytes) parameters for MsgInstallBundle"
-# See https://github.com/Agoric/agoric-sdk/blob/7b684a6268c999b082a326fdb22f63e4575bac4f/packages/agoric-cli/src/chain-config.js#L66
-RPC_MAX_BODY_BYTES=15000000
-MAX_HEADER_BYTES=$((RPC_MAX_BODY_BYTES / 10))
-MAX_TXS_BYTES=$((RPC_MAX_BODY_BYTES * 50))
-sed -i -e "s/max_body_bytes = .*/max_body_bytes = $RPC_MAX_BODY_BYTES/g" $CHAIN_DIR/config/config.toml
-sed -i -e "s/max_header_bytes = .*/max_header_bytes = $MAX_HEADER_BYTES/g" $CHAIN_DIR/config/config.toml
-sed -i -e "s/max_txs_bytes = .*/max_txs_bytes = $MAX_TXS_BYTES/g" $CHAIN_DIR/config/config.toml
-sed -i -e "s/max_tx_bytes = .*/max_tx_bytes = $RPC_MAX_BODY_BYTES/g" $CHAIN_DIR/config/config.toml
-sed -i -e "s/^rpc-max-body-bytes =.*/rpc-max-body-bytes = $RPC_MAX_BODY_BYTES/" $CHAIN_DIR/config/app.toml
-
```

Comment:
This edit was the only motivation for checking in `update-config.sh` in the first place. We should be able to delete this file and the remove the reference from config.yaml:
https://github.com/Agoric/agoric-sdk/commit/ebeb97523d62c89eda508e04437f152d41b923ae

Reply:
Noting to make that clearer next time, maybe with a comment like "This file only exists to support this change. If we find this is not needed, please remove this entire file"